crates/nu_plugin_nu_example/nu_plugin_nu_example.nu

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env -S nu --stdin
```

Comment:
A shebang makes me wonder about this working with Windows.

Reply:
I guess it works since the CI is green. yeehaw!

Reply:
Handled specifically by the plugin case along with other extensions. The shebang actually doesn't matter